spec/helpers.rb

```diff
@@ -11,4 +11,8 @@ def json
   def auth_headers
     @auth_headers ||= user.create_new_auth_token
   end
+
+  def clear_hero
```

Comment:
Maybe a better name like `restart_unique_generator` or something like that?

Reply:
ok, thanks for suggestion